e2e/src/usecases/DappListRecent.js

```diff
@@ -28,7 +28,8 @@ export default DappListRecent = () => {
 
     // Scroll to first dapp or next after most recent dapp
     await scrollToDapp(startRecentDappCount + 1)
-    await element(by.id('DappCard')).atIndex(startRecentDappCount).tap()
+    // Tap impact Market as it is the second dapp in the list
```

Comment:
Why is it always the case?

Reply:
The e2e test wallet is set to always receive the control in Statsig: [console](https://console.statsig.com/4plizaPmWwPL21ASV4QAO0/experiments/dapps_shuffle/setup), but a new dapp could be added that would take it's place.

Reply:
Thanks, would be nice to leave this as a comment in the code.

Reply:
Added a comment in https://github.com/valora-inc/wallet/pull/3937/commits/b9d876920c99651a0d320038e33893a282057731!